app/src/main/java/com/einmalfel/podlisten/Provider.java

```diff
@@ -14,6 +14,12 @@
 
 
 public class Provider extends ContentProvider {
+
+  public static final String CODE = ". Code ";
```

Comment:
1 Variables and constants are placed below inner classes in this source file
2 Maybe format string would be a cleaner solution, like this:
String WRONG_QUERY_URI_MESSAGE = "Wrong query uri %s, code: %d";